### PR TITLE
N23: Add opencombat game

### PR DESCRIPTION
**Pull request by buxx**
Add Open Combat game update.

_Part of #677_

### CODE REVIEW

**Comment by buxx** [comment]
Hello, it is my first PR ! I don't know how check mardown lint for now :(

File: content/news/023/index.md [hunk 1]
```diff
@@ -64,6 +64,28 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Game Updates
 
+### [Open Combat][opencombat]
+
+![alt text](open_combat.jpg)
```
Comment (17cupsofcoffee) [inline content/news/023/index.md:69]:
```suggestion
![Open Combat logo](open_combat.jpg)
```

File: content/news/023/index.md [hunk 2]
```diff
@@ -64,6 +64,28 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+
+[Open Combat](https://opencombat.bux.fr/)
+is a real time tactical game directly inspired from
+[Close Combat Series](https://en.wikipedia.org/wiki/Close_Combat_(series)).
+The player take control of soldier units and order them to win the battles
```
Comment (17cupsofcoffee) [inline content/news/023/index.md:74]:
```suggestion
The player takes control of soldier units and orders them to win the battles
```

File: content/news/023/index.md [hunk 3]
```diff
@@ -64,6 +64,28 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+from a top down 2D view.
+
+Project recently started with Rust language after a Python language proof of
+concept in 2017. You can find a
```
Comment (17cupsofcoffee) [inline content/news/023/index.md:78]:
```suggestion
The project recently started with Rust language after a Python language proof of
concept in 2017. More info is available at [opencombat.bux.fr][opencombat],
and you can find a
```

File: content/news/023/index.md [hunk 4]
```diff
@@ -64,6 +64,28 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+[presentation](https://www.youtube.com/watch?v=pxVgjBKXlIw) video and a game
+making of [video](https://www.youtube.com/watch?v=3TOEZ7krhvI).
```
Comment (17cupsofcoffee) [inline content/news/023/index.md:80]:
```suggestion
making of [video](https://www.youtube.com/watch?v=3TOEZ7krhvI) on YouTube.
```

File: content/news/023/index.md [hunk 5]
```diff
@@ -64,6 +64,28 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+
+_Discussions: [r/OpenCombatGame](https://www.reddit.com/r/OpenCombatGame/),
+[Discord](https://discord.gg/YD2V7XsBQZ),
+[Forum](https://discourse.opencombat.bux.fr/)_,
+Website: [opencombat.bux.fr](https://opencombat.bux.fr)
```
Comment (17cupsofcoffee) [inline content/news/023/index.md:85]:
```suggestion
[Forum](https://discourse.opencombat.bux.fr/)_
```

**Review by 17cupsofcoffee — COMMENTED** [review]
Made some minor formatting/grammar tweaks, but looks pretty much in line with what we want :)

**Review by 17cupsofcoffee — APPROVED** [review]

**Comment by buxx** [comment]
Nice ! Thanks you.

**Review by ozkriff — APPROVED** [review]
:+1:  (made a few small style tweaks: the discussions are supposed to be links to already happened threads with comments)

**Comment by ozkriff** [comment]
Merged, thanks for the PR!

**Pull request merged**